appveyor.yml

```diff
@@ -1,3 +1,10 @@
-version: 1.2.{build}
-build:
-  verbosity: minimal
\ No newline at end of file
+version: 2.0.{build}
+image: Visual Studio 2017 Preview
+
+before_build:
+- dotnet --info
+- dotnet restore -v Minimal --no-cache
+build_script:

```

Comment:
If you want automate, you can extend the after_build step to publish the package.